### PR TITLE
Clarify descriptions of APM indicators for SLOs

### LINKED ISSUE #3913: Clarify indicator for APM availability SLO
In https://www.elastic.co/guide/en/observability/current/slo-create.html#apm-latency-and-availability-sli the latency threshold is described as:

> Create an indicator based on the APM data that you received from your instrumented services and a latency threshold.

clearly indicating latency as the metric.  I expect omission of the exact field as currently implemented is deliberate to keep things simple, that seems fine.  Even so, the availability description could be more specific, currently:

> Create an indicator based on the APM data received from your instrumented services.

availability should be specific about using the transaction outcome (success/failure) as the indicator, and perhaps a note about the impact of unknown outcomes.

### PR DESCRIPTION
**Pull request by dedemorton**
Closes #3913 

@graphaelli @grabowskit WDYT? Is this what you had in mind?

If you want me to add a note about the impact of unknown outcomes, I can do that, but I need to know what the impact is. :-)

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _co-preview-docs_ successfully!

* https://docs-elastic-1nzgtcqtc-elastic-dev.vercel.app

* Built with commit 1e57c9b33751074d735b2f95b7f57978377a55bc

> Issues? Visit #next-docs in Slack

**Comment by dedemorton** [comment]
bump...I need a technical review before I can merge this. Thanks!

File: docs/en/observability/slo-create.asciidoc [hunk 1]
```diff
@@ -139,15 +139,16 @@ When defining a histogram metric SLI, set the following fields:
...
 
 *Example:* You can define an indicator on an APM service named `banking-service` for the `production` environment, and the transaction name `POST /deposit` with a latency threshold value of 300ms.
 
 [discrete]
 [[apm-availability-sli]]
 === APM availability
 
-Create an indicator based on the APM data received from your instrumented services.
+Create an indicator based on the availability of your instrumented services.
+Availability is determined by calculating the percentage of successful transactions (`event.outcome : "success"`) out of total transactions.
```
Comment (graphaelli) [inline docs/en/observability/slo-create.asciidoc:151]:
```suggestion
Availability is determined by calculating the percentage of successful transactions (`event.outcome : "success"`) out of total transactions the succeeded or failure - unknown outcomes are excluded.
```

Comment (graphaelli) [reply docs/en/observability/slo-create.asciidoc:151]: looks great!

File: docs/en/serverless/slos/create-an-slo.mdx [hunk 2]
```diff
@@ -144,11 +144,12 @@ When defining a histogram metric SLI, set the following fields:
...
 There are two types of SLI you can create based on services using application performance monitoring (APM): APM latency and APM availability.
 
-Use **APM latency** to create an indicator based on the APM data that you received from your instrumented services and a latency threshold.
+Use **APM latency** to create an indicator based on latency data received from your instrumented services and a latency threshold.
 
 **Example:** You can define an indicator on an APM service named `banking-service` for the `production` environment, and the transaction name `POST /deposit` with a latency threshold value of 300ms.
 
-Use **APM availability** to create an indicator based on the APM data received from your instrumented services.
+Use **APM availability** to create an indicator based on the availability of your instrumented services.
+Availability is determined by calculating the percentage of successful transactions (`event.outcome : "success"`) out of total transactions.
```
Comment (dedemorton) [inline docs/en/serverless/slos/create-an-slo.mdx:152]:
```suggestion
Availability is determined by calculating the percentage of successful transactions (`event.outcome : "success"`) out of the total number of successful and failed transactions—unknown outcomes are excluded.
```

**Comment by bmorelli25** [comment]
run docs-build

**Review by bmorelli25 — APPROVED** [review]

**Review by graphaelli — APPROVED** [review]

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-4odim5nro-elastic-dev.vercel.app

* Built with commit 5abae2a10e21d327ae666a5b7332c76f11dbb766

> Issues? Visit #next-docs in Slack

**Pull request merged**